FireCARES/firecares
# Feature/spam filtering

## Features:

- Adds spam filtering for the "Contact Us" section.



## Related Issues

- Closes https://github.com/FireCARES/firecares/issues/505

## Testing steps:

1. Edit `base.py`
Use this key to test:
`RECAPTCHA_SECRET = '6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe'` 
(from https://developers.google.com/recaptcha/docs/faq)
2. Go to Contact Us
3. Enter a spammy comment. This comment should not be sent to the admin 
4. Enter a normal comment. This comment should arrive at admin.
